<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myForm</class>
 <widget class="QMainWindow" name="myForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="miniWidget" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action0_1">
   <property name="text">
    <string>0.1</string>
   </property>
  </action>
  <action name="action0_2">
   <property name="text">
    <string>0.2</string>
   </property>
  </action>
  <action name="action0_5">
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="action0_8">
   <property name="text">
    <string>0.8</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1.2</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action1_8">
   <property name="text">
    <string>1.8</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>2.2</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>2.5</string>
   </property>
  </action>
  <action name="action2_8">
   <property name="text">
    <string>2.8</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3.2</string>
   </property>
  </action>
  <action name="action3_5">
   <property name="text">
    <string>3.5</string>
   </property>
  </action>
  <action name="action3_8">
   <property name="text">
    <string>3.8</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action4_3">
   <property name="text">
    <string>4.2</string>
   </property>
  </action>
  <action name="action4_5">
   <property name="text">
    <string>4.5</string>
   </property>
  </action>
  <action name="action4_8">
   <property name="text">
    <string>4.8</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
